android/src/main/java/com/novoda/notils/devicedetection/DeviceAndroidVersion.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.notils.devicedetection;
+
+import android.os.Build;
+
+public class DeviceAndroidVersion {
```

Comment:
`AndroidVersion`? it's not tied to the device. Same for the field, and then syncing up the class name and the field name (including `os` or omitting it)